python/hsfs/feature_group.py

```diff
@@ -284,3 +333,43 @@ def partition_key(self, new_partition_key):
     @online_enabled.setter
     def online_enabled(self, new_online_enabled):
         self._online_enabled = new_online_enabled
+
+    @property
+    def statistics_config(self):
+        return self._statistics_config
+
+    @statistics_config.setter
+    def statistics_config(self, statistics_config):
+        if isinstance(statistics_config, StatisticsConfig):
+            self._statistics_config = statistics_config
+        elif isinstance(statistics_config, dict):
+            self._statistics_config = StatisticsConfig(**statistics_config)
+        elif isinstance(statistics_config, bool):
+            self._statistics_config = StatisticsConfig(statistics_config)
+        elif statistics_config is None:
+            self._statistics_config = StatisticsConfig()
+        else:
+            raise TypeError(
+                "The argument `statistics_config` has to be `None` of type `StatisticsConfig, `bool` or `dict`, but is of type: `{}`".format(
+                    type(statistics_config)
+                )
+            )
+
+    @property
+    def statistics(self):
+        return self._statistics_engine.get_last(self)
+
+    def get_statistics(self, commit_time=None):
+        """Returns the statistics for this feature group at a specific time.
+
+        If `commit_time` is `None`, the most recent statistics are returned.
+
+        :param commit_time: Commit time in the format `YYYYMMDDhhmmss`, defaults to None
+        :type commit_time: str, optional
+        :return: Statistics information
+        :rtype: Statistics
+        """
+        if commit_time is None:
+            return self.statistics
+        else:
+            return self._statistics_engine.get(self, commit_time)
```

Comment:
I'd return the statistics content here, rather than the statistics object. Or at least the `to_dict()`. I doubt people are interested in the remaining fields.

Reply:
I am returning a `Statistics` object here, the same way as I do in the Java part. The statistics object only has the other fields like href as private members, and the object has no properties/getters for those fields. I can also get rid of the fields entirely.

Reply:
Ok